internal/Internal/Prelude.hs

```diff
@@ -0,0 +1,86 @@
+module Internal.Prelude
```

Comment:
So the `internal/Internal` is required because you can't call a module `Prelude` huh?

Reply:
We could mix-in the base `Prelude` under a different alias, but this approach seemed more clear.